### DISCUSSION TITLE
Enhancement: Remove Jquery dependency & merge with vanilla semantic

### DISCUSSION THREAD

**Discussion by rishavs** [fomantic/Fomantic-UI#20] [General] [answered]
Hi

Considering that we already have another fork of SUI where the main idea was to remove jquery dependency and that removing jquery seems to be one of the most requested features in SUI, i believe it will benefit the community the most if both projects were to be merged.
https://www.npmjs.com/package/vanilla-semantic-ui

**Comment by ColinFrick** [comment]
> Fomantic was created to continue active development of Semantic-UI and has the intent to be merged back into the master repository once active development can restart.

**Comment by y0hami** [comment]
As @ColinFrick mentioned this is to continue development of Semantic-UI not to create our own version of it.

**Comment by binkley** [comment]
When you do decide to cut the umbilical cord (that is, if Semantic UI does not pick back up, and Fomantic becomes the main version of this wonderful library), I would really like to see jQuery dependency go away.

**Comment by lubber-de** [comment, marked as answer]
@binkley Please take a look at https://github.com/fomantic/Fomantic-UI/issues/319 :slightly_smiling_face:
